applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/prj.conf

```diff
@@ -80,7 +80,7 @@ CONFIG_EI_WRAPPER_DATA_BUF_SIZE=1000
 
```

Comment:
Please add complete commit message (refer to commits merged to main for reference) and squash the commits before merging.